.github/label-actions.yml

```diff
@@ -274,3 +274,25 @@
 
 
     Thanks, the Renovate team
```

Comment:
The two new comments should have the `Thanks, the Renovate team` ending as well. 😉